src/common/exceptions/generic-server.exception.ts

```diff
@@ -6,10 +6,13 @@ export class GenericServerException extends Error {
     readonly status: number,
     message: string | undefined,
     readonly requestID: string,
+    readonly err: string | undefined,
+    readonly errDescription: string | undefined,
   ) {
     super();
     if (message) {
-      this.message = message;
+      this.message =
+        message + `\nError: ${err}\nError Description: ${errDescription}`;
```

Comment:
We should probably only include the `err` and `errDescription` values if they have a value, because right now we have `undefined` coming through in the resulting error message.